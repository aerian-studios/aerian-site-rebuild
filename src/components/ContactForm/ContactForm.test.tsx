import * as React from "react";

import * as renderer from "react-test-renderer";

import { ContactForm } from "./index";

describe("ContactForm", () =>
    it("renders correctly", () => {
        const tree = renderer
            .create(<ContactForm className="myClass" />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    }));
