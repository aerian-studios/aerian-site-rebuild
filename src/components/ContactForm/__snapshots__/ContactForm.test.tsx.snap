// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactForm renders correctly 1`] = `
<form
  className=" myClass"
  data-netlify={true}
  name="contact"
  onSubmit={[Function]}
>
  <div>
    <label
      htmlFor="name"
    >
      Name
    </label>
    <input
      name="name"
      onChange={[Function]}
      type="text"
      value=""
    />
  </div>
  <div>
    <label
      htmlFor="email"
    >
      Email
    </label>
    <input
      name="email"
      onChange={[Function]}
      type="email"
      value=""
    />
  </div>
  <div>
    <label
      htmlFor="phone"
    >
      Telephone
    </label>
    <input
      name="phone"
      onChange={[Function]}
      type="text"
      value=""
    />
  </div>
  <div>
    <label
      htmlFor="message"
    >
      Message
    </label>
    <textarea
      name="message"
      onChange={[Function]}
      value=""
    />
  </div>
  <input
    name="form-name"
    type="hidden"
    value="contact"
  />
  <div
    data-netlify-recaptcha={true}
  />
  <span
    className="undefined"
  >
    <button
      type="submit"
    >
      Submit
    </button>
    <i
      className="fa"
    />
  </span>
</form>
`;
